### DISCUSSION TITLE
math font size too small

### DISCUSSION THREAD

**Discussion by nschloe** [community/community#20936] [Other Feature Feedback, Questions, & Ideas]
GitHub's math font size is chosen to match the height of the capital letters of the surrounding text. This makes the small letters in math mode smaller than their text counterpart, leading to hardly readable fractions/exponents/indices:

<img src='https://user-images.githubusercontent.com/181628/179930526-4ad8c827-221b-4402-94d5-1a0e1b5dc28f.png' width="25%"/>

Other pages use larger font sizes. Examples:

- [math.stackexchange.com](https://math.stackexchange.com/)

  ![screenshot](https://user-images.githubusercontent.com/181628/179928699-3400e95e-a046-4058-ac9b-18c242475d23.png)

- [Wikipedia](https://en.wikipedia.org/wiki/Cauchy%27s_integral_theorem#Formulation_on_Simply_Connected_Regions):

  ![screenshot](https://user-images.githubusercontent.com/181628/180648584-00ea0b67-873b-4733-8208-9a4e9e81dd56.png)

- [mathoverflow](https://mathoverflow.net/questions/34059/if-f-is-infinitely-differentiable-then-f-coincides-with-a-polynomial):

  ![screenshot](https://user-images.githubusercontent.com/181628/180649089-2635b435-8196-4e5d-a710-852c1111402c.png)

On GitHub:

![screenshot](https://user-images.githubusercontent.com/181628/179928903-305c3ed0-6c19-47e8-863f-dc2d9cbc525c.png)

I would hence like to suggest choosing a larger font size for math.

**Comment by MrSarno** [comment]
I understand this isn't a proper solution to the issue you raise, however in the short-term you could always use the browser plugin Stylish ([Firefox](https://addons.mozilla.org/en-US/firefox/addon/stylish/) / [Chrome & Chromium-based](https://chrome.google.com/webstore/detail/stylish-custom-themes-for/fjnbnpbmkenffdnngjfgmeleoegfcffe?hl=en)) to manually adjust how the math on GitHub is displayed in your browser.

If you'll forgive the somewhat lazy example, creating the following rule;

> .js-inline-math {
>     font-size: 20px !important;
> }

and having it applied to pages beginning with `https://github.com` might go some way towards alleviating this.

<img width="716" alt="Screenshot 2022-07-20 at 18 39 59" src="https://user-images.githubusercontent.com/32073555/180047409-5e0e2b97-2d05-4382-840d-c5bc947d86f4.png">

**Comment by michaeljoyy852-del** [comment]
O tamanho da fonte matemática está muito pequeno, o que dificulta bastante a leitura e compreensão do conteúdo. Seria ótimo aumentar o tamanho para melhorar a experiência do usuário. Isso é especialmente importante para quem gosta de usar [letras diferentes](fontesletrasdiferentes.com.br), pois a visualização clara faz toda a diferença.
